### ISSUE TITLE
 I am training with my own data set, but the problem of loss is empty.

### ISSUE THREAD

**Issue by dazhangzhang** [amdegroot/ssd.pytorch#300]
my pc 
GTX1080Ti  centos7   learnng rate 1e-5
Who has the same situation as me, please help me. thanks。
 iter 310 || Loss: nan || timer: 0.1921 sec.
iter 320 || Loss: nan || timer: 0.2041 sec.
iter 330 || Loss: nan || timer: 0.2006 sec.
iter 340 || Loss: nan || timer: 0.2043 sec.
iter 350 || Loss: nan || timer: 0.2128 sec.
iter 360 || Loss: nan || timer: 0.2072 sec.
iter 370 || Loss: nan || timer: 0.2091 sec.
iter 380 || Loss: nan || timer: 0.2141 sec.
iter 390 || Loss: nan || timer: 0.2486 sec.
iter 400 || Loss: nan || timer: 0.1914 sec.
iter 410 || Loss: nan || timer: 0.2052 sec.
iter 420 || Loss: nan || timer: 0.1976 sec.
iter 430 || Loss: nan || timer: 0.1952 sec.
iter 440 || Loss: nan || timer: 0.1942 sec.
iter 450 || Loss: nan || timer: 0.2101 sec.
iter 460 || Loss: nan || timer: 0.1934 sec.

**Comment by tengshaofeng** [comment]
@dazhangzhang , i also miss the same problem. have you solved that?

**Comment by dazhangzhang** [comment]
@amdegroot   please   can you help me

**Comment by wisdomk** [comment]
try to change the learning rate

**Comment by JappaB** [comment]
Indeed, lower batch-size needs a lower learning rate. If you didn't change the default parameters it should work though..

**Comment by HanningLin** [comment]
I used --lr 5e-4 and defult batch_size. It seems worked out.

**Comment by YunjieYu** [comment]
I solved this problem by setting num_workers = 0 (0 means that the data will be loaded in the main process.). It seems that using many subprocesses  for data loading could lead to loss-nan problem.
